src/main-oe/sgxlkl_run_oe.c

```diff
@@ -1823,7 +1759,7 @@ int main(int argc, char* argv[], char* envp[])
     {
         get_signed_libsgxlkl_path(libsgxlkl, PATH_MAX);
     }
-    sgxlkl_host_verbose_raw("result=%s\n", libsgxlkl ? libsgxlkl : "null");
+    sgxlkl_host_verbose_raw("result=%s\n", libsgxlkl);
```

Comment:
Shouldn't we print "not found" if it couldn't locate the enclave library?

Reply:
`get_signed_libsgxlkl_path` will already exit and spit out an error.